src/java.base/share/classes/jdk/internal/vm/annotation/IntrinsicCandidate.java

```diff
@@ -28,94 +28,67 @@
 import java.lang.annotation.*;
 
 /**
- * The {@code @IntrinsicCandidate} annotation is specific to the
- * HotSpot Virtual Machine. It indicates that an annotated method
- * may be (but is not guaranteed to be) intrinsified by the HotSpot VM. A method
- * is intrinsified if the HotSpot VM replaces the annotated method with hand-written
- * assembly and/or hand-written compiler IR -- a compiler intrinsic -- to improve
- * performance. The {@code @IntrinsicCandidate} annotation is internal to the
- * Java libraries and is therefore not supposed to have any relevance for application
- * code.
+ * The {@code @IntrinsicCandidate} indicates that an annotated method is
+ * recognized by {@code vmIntrinsics.hpp} for special treatment by the HotSpot
+ * VM, unlike the other methods.  The HotSpot VM checks, when loading a class,
```

Comment:
/, unlike the other methods/s/the other/other/

If you want something at the top that teaches about "the other methods", maybe say something like:

> Normally, the VM executes methods using only their bytecodes (or JNI definitions), but the VM is (in addition) allowed to use specialized code or analysis logic for methods marked as intrinsics.  Thus, intrinsic calls can be replaced or modified or specially optimized by the VM.

But those points will be covered further down, so maybe not.

More concisely:  s/, unlike the other methods/, unlike normal methods, which depend only on bytecodes or native entry points for their meaning to the VM/

Or just: s/, unlike the other methods//